pages/10.contacts/03.manage-segments/docs.en.md

```diff
@@ -143,6 +143,14 @@ manually added will remain part of the list regardless of filters.
 In addition to segments you can also manually add any contact to a list by
 clicking the Preferences button at the segments tab, use the dropdown to select a segment and add the contact to it or click on the x next to a segment in the input field to remove the contact.
 
+## Add categories to segment
+
+Apart from organizing various Mautic elements, categories help with organizing contacts. To add categories to a segment, create a new [category](../categories#creating-managing-categories) of type `segment`. When you add a new segment, you can now sort and select from the categories you previously created.
+
+![Screenshot showing segment categories](add-category-segment.jpg)
```

Comment:
Please can you use a rectangle rather than freehand drawing, so that it looks more professional?

If you need a tool for that I use the Nimbus Screenshot extension for Chrome which is pretty easy to use!